jetbrains-core/src/software/aws/toolkits/jetbrains/core/utils/CollectionUtils.kt

```diff
@@ -7,6 +7,7 @@ package software.aws.toolkits.jetbrains.core.utils
  * Replace with [kotlin.collections.buildList] when experimental is removed
  */
 inline fun <E> buildList(builderAction: MutableList<E>.() -> Unit): List<E> = ArrayList<E>().apply(builderAction)
+inline fun <E> buildList(capacity: Int, builderAction: MutableList<E>.() -> Unit): List<E> = ArrayList<E>(capacity).apply(builderAction)
```

Comment:
Is this used anywhere?

Reply:
Probably not anymore, I was using it at one point to build the Index lists, but dropped it

Reply:
Still used in IndexComboBoxModel